CMakeLists.txt

```diff
@@ -1,30 +1,50 @@
-cmake_minimum_required(VERSION 3.0)
-
-project(umesimd)
-
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/doc)
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/examples)
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/plugins)
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/utilities)
-
-file(GLOB top_files
-        UMEBasicTypes.h
-        UMEInline.h
-        UMEMemory.h
-        UMESimd.h
-        UMESimdInterface.h
-        UMESimdInterfaceFunctions.h
-        UMESimdScalarEmulation.h
-        UMESimdScalarOperators.h
-        UMESimdTraits.h
-        UMESimdVectorEmulation.h
-        README.md
-        LICENSE)
-
-install(DIRECTORY doc DESTINATION include/ume/simd/doc)
-install(DIRECTORY examples DESTINATION include/ume/simd/examples)
-install(DIRECTORY plugins DESTINATION include/ume/simd/plugins)
-install(DIRECTORY utilities DESTINATION include/ume/simd/utilities)
-install(FILES ${top_files} DESTINATION include/ume/simd)
-
-include(CPack)
\ No newline at end of file
+cmake_minimum_required(VERSION 3.1)
```

Comment:
I will check for the actual minimum version. I usually choose version 3.1 because of the drastic change in behavior due to policy [CMP0054](https://cmake.org/cmake/help/v3.1/policy/CMP0054.html), which is often the source of bugs.